### ISSUE TITLE
BigBrain precomputed neuroglancer data as JSON

### ISSUE THREAD

**Issue by marcenko** [FZJ-INM1-BDA/siibra-api#62]
At the moment we have a link to get precomputed neuroglancer data as **application/octet-stream**.

Example - see:

`https://siibra-api-latest.apps-dev.hbp.eu/v1_0/atlases/juelich%2Fiav%2Fatlas%2Fv1.0.0%2F1/spaces/minds%2Fcore%2Freferencespace%2Fv1.0.0%2Fa1655b99-82f1-420f-a3c2-fe80fd4c8588`

`https://neuroglancer.humanbrainproject.eu/precomputed/BigBrainRelease.2015/8bit`

We should provide a way to get the data as **application/json**.

**Comment by xgui3783** [comment]
hmm

```sh
curl -v 'https://siibra-api-latest.apps-dev.hbp.eu/v1_0/atlases/juelich%2Fiav%2Fatlas%2Fv1.0.0%2F1/spaces/minds%2Fcore%2Freferencespace%2Fv1.0.0%2Fa1655b99-82f1-420f-a3c2-fe80fd4c8588'
```

already returns `application/json` as content type

**Comment by xgui3783** [comment]
this issue is resolved as of v0.3.17 (`/map/` endpoint returns structured JSON)
